### PR TITLE
1299 double dataization

### LINKED ISSUE #1299: lt dataizes twice
@Graur please take a look at this case
```
[] > less-than
  assert-that > @
    seq
      QQ.io.stdout "SEQ STARTING\n"
      100
    $.less-than 300
```
This code works this way
```
[INFO] Running EOorg.EOeolang.EOthreads.EOless_thanTest
SEQ STARTING
SEQ STARTING
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.012 s - in EOorg.EOeolang.EOthreads.EOless_thanTest
```
I don't guess it must work this way.

### PR DESCRIPTION
**Pull request by includealex**
Closes #1299

### CODE REVIEW

**Comment by includealex** [comment]
@Graur please take a look.

**Comment by includealex** [comment]
@levBagryansky please take a look.

**Comment by levBagryansky** [comment]
@includealex sure

**Review by levBagryansky — CHANGES_REQUESTED** [review]
@includealex Do we realy need to check `stdout`?

**Comment by levBagryansky** [comment]
@includealex May be it would be better to implemet the test in java if you want to check `stdout`? Look at [this](https://github.com/objectionary/eo/blob/master/eo-runtime/src/test/java/EOorg/EOeolang/EOintEOminusTest.java) to understand how to use eo objects in java

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ target/
 .classpath
 .recommenders
 .mvn/wrapper/maven-wrapper.jar
+.factorypath
```
Comment (Graur) [inline .gitignore:11]: @includealex Why do we need this change?

**Review by Graur — CHANGES_REQUESTED** [review]

**Comment by includealex** [comment]
@levBagryansky please take a look

**Review by levBagryansky — APPROVED** [review]
@includealex looks good as for me

**Comment by includealex** [comment]
@yegor256 please take a look

File: eo-runtime/src/test/java/EOorg/EOeolang/EOseqTest.java [hunk 2]
```diff
@@ -94,5 +94,4 @@ public void makesTrueCopy() {
             Matchers.equalTo(2L)
         );
     }
-
```
Comment (Graur) [inline eo-runtime/src/test/java/EOorg/EOeolang/EOseqTest.java:97]: @includealex Do we really need this change?

File: eo-runtime/src/test/eo/org/eolang/single_dataization-tests.eo [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+# SOFTWARE.
+
++alias org.eolang.hamcrest.assert-that
++architect yegor256@gmail.com
++home https://github.com/objectionary/eo
++junit
++package org.eolang
++version 0.0.0
+
+# @todo #1299:60min Implement test that will
```
Comment (Graur) [inline eo-runtime/src/test/eo/org/eolang/single_dataization-tests.eo:30]: @includealex this `@todo` will fail `pdd` check. Can you add an extra space between text and `#` on each line after the first line?

**Review by Graur — CHANGES_REQUESTED** [review]

**Comment by includealex** [comment]
@Graur please check it.

**Review by Graur — APPROVED** [review]
@yegor256 please, have a look

**Comment by includealex** [comment]
@yegor256 please take a look

**Comment by yegor256** [comment]
@includealex please, see how all other test files are named in `src/test/eo` and make sure your tests are placed right

**Comment by includealex** [comment]
@yegor256 please have a look

**Comment by includealex** [comment]
@yegor256 please take a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**